### ISSUE TITLE
sea lanterns blinding 

### ISSUE THREAD

**Issue by donutgoose** [espen96/Loop#22]
when I look at sea lanterns, they are unreasonably bright. How do I turn it down? please forgive if this is the wrong place I am not an avid github user

**Comment by espen96** [comment]
hi!, could you provide a screenshot for reference?

**Comment by donutgoose** [comment]
yes sure. I am not currently at home, though, so it will be a few hours

edit:
![image](https://user-images.githubusercontent.com/64748981/205790843-23844b6a-937d-4c7b-b594-470cc1d6d977.png)

**Comment by espen96** [comment]
I see,sadly there isn't any way to adjust that.
 I'll take your feedback into consideration in the future tho
